packages/react/src/Card/Card.features.stories.tsx

```diff
@@ -44,6 +44,25 @@ export const CTAText: StoryFn<typeof Card> = () => {
   )
 }
 
+export const FullWidth: StoryFn<typeof Card> = () => {
```

Comment:
Is it worth adding the `fullWidth` prop to the Card playground story too?

Reply:
💯 thanks for reminder @joshfarrant... totally missed that. Would be nice if auto args just worked though 🤷